cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,31 @@
-describe('', () => {
-  before(() => {
+describe('Conduit', () => {
+    before(() => {
+      cy.clearCookies(); 
+      cy.clearLocalStorage();
+      cy.visit('/user/login');
+      cy.get('input[placeholder="Email"]').should('be.visible');
+      cy.get('input[placeholder="Email"]').type('hogwortsexpress@gmail.com');
+      cy.get('input[placeholder="Password"]').type('Qwert123!');
+      cy.get('.btn').click();
```

Comment:
It's a good practice to move the login functionality to a common function or a custom cypress command. This way, you can reuse the login functionality across different test cases without duplicating the code.